frontend/src/App.js

```diff
@@ -16,13 +16,16 @@ import Camera from "./components/Camera";
 import Words from "./components/Words";
 
 import Buttons from "./components/Buttons";
-const song = new Audio("/songs/Daylight.mp3");
-song.volume = 0.01;
-const toneArmOnSoundeffect = new Audio("/soundeffects/tonearm_on_sound.mp3");
-toneArmOnSoundeffect.volume = 0.4;
-const vinylSoundeffect = new Audio("/soundeffects/vinyl_soundeffect.mp3");
-vinylSoundeffect.volume = 1;
-vinylSoundeffect.loop = true;
+import Song from "./components/Song";
+import Lights from "./components/Lights";
+
+// let song = new Audio("/songs/Daylight.mp3");
+// song.volume = 0.01;
+// const toneArmOnSoundeffect = new Audio("/soundeffects/tonearm_on_sound.mp3");
+// toneArmOnSoundeffect.volume = 0.4;
+// const vinylSoundeffect = new Audio("/soundeffects/vinyl_soundeffect.mp3");
+// vinylSoundeffect.volume = 1;
+// vinylSoundeffect.loop = true;
```

Comment:
Do you want to delete this if it's commented or are you going to keep it for historical reasons?

Reply:
Will delete